src/main/java/io/cdap/plugin/kudu/KuduSinkConfig.java

```diff
@@ -260,4 +317,139 @@ public int getSeed() {
   public int getThreads() {
     return (optBossThreads != null) ? Integer.parseInt(optBossThreads) : 1;
   }
+
+  public void validate(FailureCollector failureCollector) {
+    if (Strings.isNullOrEmpty(optSchema)) {
+      failureCollector.addFailure("Write schema is not specified.", null)
+        .withConfigProperty(SCHEMA);
+    } else {
+      try {
+        Schema.parseJson(optSchema);
+      } catch (IOException e) {
+        failureCollector.addFailure( String.format("Unable to parse schema '%s'", optSchema),
+                                     "Provide correct schema.")
+          .withStacktrace(e.getStackTrace())
+          .withConfigProperty(SCHEMA);
+      }
+    }
+
+    // If there is macro specified for 'master' address or table name, then
+    // we defer the creation of table to initialize.
+    if (!containsMacro(MASTER)) {
+      if (Strings.isNullOrEmpty(optMasterAddresses)) {
```

Comment:
remove this check. We should control the value being present with adding or not adding @nullable.

Reply:
Done